src/Algebra/Morphism/Construct/DirectProduct.agda

```diff
@@ -107,3 +127,24 @@ module Monoid (M : RawMonoid a ℓ₁) (N : RawMonoid b ℓ₂) where
       where
         module F = IsMonoidHomomorphism F
         module H = IsMonoidHomomorphism H
+
+-- Package for export
+module Monoid-Export {M : RawMonoid a ℓ₁} {N : RawMonoid b ℓ₂} where
+  open Monoid
+
+  private
+    module M = RawMonoid M
+    module N = RawMonoid N
+
+  module _ {refl : Reflexive M._≈_} where
+    proj₁ = Proj₁.isMonoidHomomorphism M M refl
+
+  module _ {refl : Reflexive N._≈_} where
+    proj₂ = Proj₂.isMonoidHomomorphism M N refl
+
+  module _ {P : RawMonoid c ℓ₃} where
+
+    private
+      module P = RawMonoid P
```

Comment:
It doesn't look like you are using this below, you are using the record, not the module, no?

Reply:
This was an overlook